demo/app/app.component.ts

```diff
@@ -41,6 +43,11 @@ export class AppComponent implements AfterViewInit {
 
   ngAfterViewInit(): void {
     // example for getting the viewer by Id outside of the ac-map hierarchy
-    const viewer = this.viewersManager.getMap('main-map');
+    const map = this.mapsManagerService.getMap('main-map');
+    const viewer = map.getCesiumViewer();
+
+    const coordinateConverter = map.getCoordinateConverter();
+    this.draggableToMapService.init(coordinateConverter);
```

Comment:
user needs to provide the coordinate converter?
maybe `draggableToMapService` will create his own `coordinateConverter` using this.mapsManagerService.getMap() default map, if the user has a number of maps he will be able to pass map-id.

Reply:
ok